ip_map_branch.txt

```diff
@@ -17,8 +17,8 @@ four_i	https://github.com/bradymiller/openemr.git	rebase-lifemesh_telehealth_int
 five	https://github.com/openemr/openemr.git	v6_0_0_2	0	0	0	0	demo_5_0_0_5.sql	0	1	demo.openemr.io	hey	tag	5.0.0	0	4	0	Public OpenEMR 6.0.0 Production Demo
 five_a	https://github.com/openemr/openemr.git	v6_0_0_2	0	0	0	0	demo_5_0_0_5.sql	0	1	demo.openemr.io/a	hey	tag	5.0.0	0	4	0	Alternate Public OpenEMR 6.0.0 Production Demo
 five_b	https://github.com/openemr/openemr.git	v6_0_0_2	0	0	0	0	demo_5_0_0_5.sql	0	1	demo.openemr.io/b	hey	tag	5.0.0	0	4	0	Another Alternate Public OpenEMR 6.0.0 Production Demo
-six	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	six.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.10
-six_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	six.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.10 With Demo Data
+six	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	six.openemr.io	hey	branch	0	2	0	0	OPEN
+six_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	six.openemr.io/a	hey	branch	OPEN
 six_b	https://github.com/openemr/openemr.git	rel-502	0	0	0	0	0	0	1	six.openemr.io/b	hey	branch	0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Alpine 3.10
```

Comment:
looks like more than just the description was changed (ie. removed) in six_a
(agree with opening this up (and then dropping https://www.open-emr.org/wiki/index.php/Development_Demo#Alternate_Demo_.28Alpine_3.10_-_uses_php_7.3.29), however note the 5.0.2 demo six_b and six_c are still needing to stay online, so will be stuck with six and six_a running anyways, so they can either load up and do nothing or load up and be considered dev demos that simply tell us the php version is too low; it's good to have one demo do this btw since we can test that php version not compatible mechanism is still working)